extensions-core/stats/src/test/java/org/apache/druid/query/aggregation/variance/sql/VarianceSqlAggregatorTest.java

```diff
@@ -698,4 +701,28 @@ public void testVarianceAggAsInput()
         expectedResults
     );
   }
+
+  @Test
+  public void testOverWindow()
+  {
+    testBuilder()
+        .sql(
+            "select dim4, dim5, mod(m1, 3), var_pop(mod(m1, 3)) over (partition by dim4 order by dim5) c\n"
+            + "from numfoo\n"
+            + "group by dim4, dim5, mod(m1, 3)")
+        .queryContext(ImmutableMap.of(
+            PlannerContext.CTX_ENABLE_WINDOW_FNS, true,
+            QueryContexts.ENABLE_DEBUG, true,
+            QueryContexts.WINDOWING_STRICT_VALIDATION, false
```

Comment:
this will break the other PR; so after one of these gets merged - to other needs updates